docs/integrations/context-and-outputs.md

```diff
@@ -139,5 +141,630 @@ return_outputs(
+- contextPath: DBotScore.Indicator
+  description: The indicator that was tested.
+  type: String
+- contextPath: DBotScore.Type
+  description: The indicator type.
+  type: String
+- contextPath: DBotScore.Vendor
+  description: The vendor used to calculate the score.
+  type: String
+- contextPath: DBotScore.Score
+  description: The actual score.
+  type: Number
+```
+
+
+
+*Markdown*
+### Results
+|asn|confidence|indicator|
+|---|---|---|
+| 12345 | 95 | 5.5.5.5 |
+
+
+#### <a name='return-domain-reputation'></a> Return Domain reputation
+```python
+domain_raw = get_domain_from_autofocus('google.com')
+
+if domain_raw.get('score') > 90:
+    score = Common.DBotScore.BAD
+elif domain_raw.get('score') > 60:
+    score = Common.DBotScore.SUSPICIOUS
+else:
+    score = Common.DBotScore.GOOD
+
+dbot_score = Common.DBotScore(
+    indicator='google.com',
+    indicator_type=DBotScoreType.DOMAIN,
+    integration_name='Autofocus v2',
+    score=score
+)
+
+domain = Common.Domain(
+    domain='google.com',
+    dns=domain_raw.get('dnssec'),
+    creation_date=domain_raw.get('creation_date'),
+    positive_detections=domain_raw.get('positive_detections'),
+    detection_engines=domain_raw.get('detection_engines'),
+    sub_domains=domain_raw.get('sub_domains'),
+    whois=Common.WHOIS(
+        domain_status=domain_raw.get('status'),
+        name_servers=domain_raw.get('name_servers'),
+        registrar_name=domain_raw.get('registrar'),
+        expiration_date=domain_raw.get('expiration_date')
+    ),
+    dbot_score=dbot_score
+)
+
+results = CommandResults(
+    outputs_prefix='Autofocus.Domain',
+    outputs_key_field='domain',
+    outputs=domain_raw,
+    indicators=[domain]
+)
+
+return_results(results)
+```
+
+*YAML Definition*
+```yaml
+# Reputation commands usually should return DBotScore object - https://xsoar.pan.dev/docs/integrations/context-standards#dbot-score
+- contextPath: DBotScore.Indicator
+  description: The indicator that was tested.
+  type: String
+- contextPath: DBotScore.Type
+  description: The indicator type.
+  type: String
+- contextPath: DBotScore.Vendor
+  description: The vendor used to calculate the score.
+  type: String
+- contextPath: DBotScore.Score
+  description: The actual score.
+  type: Number
+
+# This is standard context output - https://xsoar.pan.dev/docs/integrations/context-standards#ip
+- contextPath: Domain.Name
+  description: 'The domain name, for example: "google.com".'
+  type: String
+- contextPath: Domain.CreationDate
+  description: The date that the domain was created.
+  type: Date
+- contextPath: Domain.DNS
+  description: A list of IP objects resolved by DNS.
+  type: String
+- contextPath: Domain.WHOIS.NameServers
+  description: Name servers of the domain.
+  type: String
+- contextPath: Domain.WHOIS.Registrar.AbuseEmail
+  description: The email address of the contact for reporting abuse.
+  type: Unknown
```

Comment:
lets not have `Unknown` type in the example

Reply:
Unknown is usually used for Array/List